s3update.go

```diff
@@ -89,7 +91,11 @@ func runAutoUpdate(u Updater) error {
 		return fmt.Errorf("invalid local version")
 	}
 
-	svc := s3.New(session.New(), &aws.Config{Region: aws.String(u.S3Region)})
+	if u.AWSConfig != nil {
+		u.AWSConfig = &aws.Config{Region: aws.String(u.S3Region)}
+	}
+	svc := s3.New(session.New(), u.AWSConfig)
```

Comment:
Is it safe to pass a aws config that is `nil` ? 🤔

Reply:
oups wrong condition :smile:

Reply:
fixed